stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/member/GroupChatInfoMemberOptionsViewModel.kt

```diff
@@ -3,24 +3,27 @@ package io.getstream.chat.ui.sample.feature.chat.info.group.member
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MediatorLiveData
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.Transformations
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.viewModelScope
 import io.getstream.chat.android.client.api.models.QuerySort
 import io.getstream.chat.android.client.call.await
 import io.getstream.chat.android.client.models.Channel
 import io.getstream.chat.android.client.models.Filters
-import io.getstream.chat.android.livedata.ChatDomain
-import io.getstream.chat.android.livedata.controller.QueryChannelsController
 import io.getstream.chat.android.livedata.utils.Event
+import io.getstream.chat.android.offline.querychannels.QueryChannelsController
 import io.getstream.chat.ui.sample.common.isDraft
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.flatMapConcat
+import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
 
 class GroupChatInfoMemberOptionsViewModel(
     private val cid: String,
     private val memberId: String,
-    private val chatDomain: ChatDomain = ChatDomain.instance(),
+    private val chatDomain: io.getstream.chat.android.offline.ChatDomain = io.getstream.chat.android.offline.ChatDomain.instance(),
```

Comment:
Why do you use the full class name? Why don't just import it?

Reply:
Small mistake